Source/Charts/Utils/Platform.swift

```diff
@@ -245,9 +245,9 @@ public typealias NSUIGestureRecognizerState = UIGestureRecognizer.State
             return _timestamp
         }
 
-		init(target: AnyObject, selector: Selector)
+		init(target: Any, selector: Selector)
```

Comment:
despite the weird indent here by github, why we use Any and again  cast to AnyObject later
I assume using AnyObject is fine

Reply:
This is to align with how ObjC APIs are imported into Swift. More info here: https://developer.apple.com/swift/blog/?id=39

We were only using `AnyObject` in the initializer because of weird objc interop.

Reply:
oh no. I'm using Dash and cannot even found `Any` type. this kind of information is not easily to be aware of in my daily work..